src/v/hashing/tests/BUILD

```diff
@@ -24,3 +24,21 @@ redpanda_cc_btest_no_seastar(
         "//src/v/utils:named_type",
     ],
 )
+
+redpanda_cc_bench(
+    name = "hashing_bench",
```

Comment:
we need the binary names to match the CMake binaries for some reporting to work. This needs to be `hashing_bench_rpbench`